Csrc/prognostic_variables.h

```diff
@@ -228,9 +228,9 @@ void set_bcs(ssize_t dim, ssize_t s, double bc_factor ,struct DimStruct *dims,
                              }
                         }
                         else{
-                            values[ishift + jshift + bc_start ] = 0.0;
+                            values[ishift + jshift + bc_start -1] = 0.0;
                             for(k=1;k<dims->gw;k++){
```

Comment:
@szy21 we need to see where these changes came from - they are not clear to me at all
this is the most worrisome change I see in this PR

Reply:
This is also from the original files that Kyle sent me. I tried to read the code and it seems this line will only be applied to vertical velocity `w`, but I don't know where these changes came from.